src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Areacount,Figuredrawing {

```

Comment:
let's implement interfaces in the Figure class and remove interfaces from class declaration of circle and other figures